SixTest/stf_output/compare.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
If you could get this program to accept two command line arguments - the STF format and the non-STF format file to compare - that would be useful! It could then be used to compare the outputs of any tests by comparing the new and the old canonical. Will be very useful whenever we need to modify canonicals (after fixing bugs etc.).